api/lib/application/users/user-controller.js

```diff
@@ -332,4 +334,12 @@ module.exports = {
     });
     return h.response(userOrganizationForAdminSerializer.serialize(organizations));
   },
+
+  async findCertificationCenterMembershipsByUser(request, h) {
```

Comment:
**thought**
En voyant la méthode `findCertificationCenterMembershipsByUser`, je me suis dit "est-ce que ça n'aurait pas plutôt sa place dans un controller "certification-center-membership ?". Je trouve qu'à cet égard la formulation `findUserOrganizationsForAdmin` qu'on a au-dessus est meilleure, on comprend mieux pourquoi c'est là.

Reply:
Cette méthode est vraiment sémantiquement liée au user car on cherche les _certification centers_ rattachés à l'utilisateur. Et pas la relation inverse.

Par contre, effectivement, la notion « for admin » est absente de cette méthode.

Donc je serais plutôt pour une des deux options suivantes :

**a)** On déplace cette méthode dans un nouveau controller dédié `admin/user-controller.js` ou quelque chose du genre pour bien indiquer qu'on est dans le contexte _admin_
**b)** On laisse comme ça